packages/babylon/src/types.js

```diff
@@ -1068,6 +1068,8 @@ export type TsType =
   | TsArrayType
   | TsTupleType
   | TsUnionOrIntersectionType
+  | TsConditionalType
+  | TsInferType
```

Comment:
cc @andy-ms I assume this was just missed in your PR?

Reply:
@loganfsmyth Yep, thanks